### DISCUSSION TITLE
[FEATURE]: Cors support for specification url

### DISCUSSION THREAD

**Discussion by treii28** [etsy/open-api#708] [Ideas]
### Is there an existing ticket for this?

- [X] I have searched the existing tickets

### Is your feature request related to a problem? Please describe

Would it be possible to open up the CORS access on the specification url?  I use swagger-ui locally on the spec and it would be nice to always have an up-to-date reference by pointing it directly at your live/published specification. However, doing so fails due to the cross-origin restrictions on that url.

https://www.etsy.com/openapi/generated/oas/3.0.0.json

It may also be useful to alias that to the most current version or the most current major version. e.g.:

https://www.etsy.com/openapi/generated/oas/current.json

or

https://www.etsy.com/openapi/generated/oas/v3-current.json

(or something similar)

### Describe the solution you'd like

set CORS to allow remote access to the specification so it can be consumed by a swagger-ui installation on another website.

### Describe any alternatives you've considered

I have copied a local version of the spec for the time being, but if you update your spec

**Comment by etsyachristensen** [comment]
@treii28 We are investigating this possibility. In the meantime, we are closing the issues board, so I'll convert this to a discussion in the Ideas category for easier tracking.
